modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Manage/User/UserForm.tsx

```diff
@@ -215,19 +194,17 @@ const UserForm = () => {
 						{roles.map(({id, name}: {id: number; name: string}) => (
 							<div className="mt-2" key={id}>
 								<ClayCheckbox
-									checked={rolesWatch.includes(id)}
+									checked={userRoles.includes(id)}
+									disabled={!userRoles.includes(id)}
 									label={name}
-									onChange={(event) =>
-										setCheckedValue(event.target.value)
-									}
-									value={id}
+									onChange={() => {}}
 								/>
 							</div>
 						))}
 					</ClayLayout.Col>
 				</ClayLayout.Row>
 
-				<Form.Divider />
+				<br />
```

Comment:
No need for this, maybe should be better to put a margin bottom in <ClayLayout.Row className="mb-2"

```suggestion
			
```